tests/system_tests_http1_adaptor.py

```diff
@@ -905,7 +918,7 @@ def test_02_backpressure_server(self):
         client_sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
The leaking sockets probably don't matter too much in this test. A leaking router would be a bigger problem.

Reply:
Yeah while we don't want to leak sockets this particular set of tests re-create the router and all the ephemeral ports for each test.  That should avoid the "socket in use" errors we've seen on other tests since  in those tests the TestServers use the same port # for all tests (since the router HTTP server-facing connection configuration doesn't change for each individual sub-test).  When one of those TestServers is left running from a previous test things get ugly.